src/nuget/Microsoft.DotNet.BuildTools.nuspec

```diff
@@ -20,6 +20,7 @@
       <dependency id="Microsoft.DotNet.TestHost" version="1.0.3-prerelease" />
       <dependency id="Microsoft.DotNet.CoreCLR" version="1.0.0-prerelease" />
       <dependency id="Microsoft.DotNet.BuildTools.ApiTools" version="1.0.0-prerelease" />
+      <dependency id="NuProj" version="0.10.4-beta-gf7fc34e7d8" />
```

Comment:
Thanks - I wasn't seeing the change when I tried to view the history of the nuspec file through this PR, but see it now.  I'll update to include the NuProj binary and targets in this package.